common/djangoapps/util/user_utils.py

```diff
@@ -0,0 +1,18 @@
+"""
```

Comment:
I believe we are deprecating the common/djangoapps directory in favor of openedx/core/djangoapps.